### DISCUSSION TITLE
Welcome to MeshWalker Discussions!

### DISCUSSION THREAD

**Discussion by AlonLahav** [AlonLahav/MeshWalker#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by lishawa** [comment]
Thanks for sharing the codes. I have some confusion when evaluate human body segmentation using the downloaded pretrained model, since the metrics provided in evaluation_segmentation.py seem to be edge accuracy and vertex accuracy,  not face accuracy (92.7%) reported in the paper. Could you kindly share me the codes for the calculation of face accuracy (92.7%) on the original meshes?
